webpack.config.js

```diff
@@ -16,6 +20,8 @@ const babel = require('babel-core');
 const babelLoaderVersion =
   require('./node_modules/babel-loader/package.json').version;
 
+const isProduction = process.env.NODE_ENV === 'production';
```

Comment:
So, I just had a thought—`yargs` presumably is just looking at the arguments passed to the current process when it started. That information should be no different in a `webpack.config` module that’s `require`d into the `gulpfile` than it is in the `gulpfile` itself, so could we just make this line `const isProduction = yargs.argv.production` and roll back some of the other related changes?